src/ts/components/seekbar.ts

```diff
@@ -220,7 +220,9 @@ export class SeekBar extends Component<SeekBarConfig> {
           this.setPlaybackPosition(100);
         } else {
           let playbackPositionPercentage = 100 - (100 / player.getMaxTimeShift() * player.getTimeShift());
-          this.setPlaybackPosition(playbackPositionPercentage);
+          if (!this.isSeeking()) {
```

Comment:
i guess the calculation of the `playbackPositionPercentage` can also be moved into this if, since it's unused outside?